### PR TITLE
feat(test): Add local homebrew test

### PR DESCRIPTION
**Pull request by revilowaldow**
```js
const fileList = Uf.listJsonFiles("homebrew").filter(item => item !== "homebrew/index.json");
```
ehhhhh, better would be to read the local index and then check the listed files, unsure how this would behave if there is a mix of remote and local links in an index
I also couldn't get `blocklistDir` to accept a single file so moved on.

The index itself could also be tested, though I couldn't be bothered doing a PR here and to the utils repo to add a schema there.
Getting the json in the index right is trivial, it wouldn't check the links which would be the helpful bit

### CODE REVIEW

File: test/test-brew.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	const results = await jsonTester.pGetErrorsOnDirsWorkers({
+		isFailFast: !_IS_FAIL_SLOW,
+		fileList,
+	});
+
+	const {errors, errorsFull} = results;
+
+	if (errors.length) {
+		if (!process.env.CI) fs.writeFileSync(`test/test-json.error.log`, errorsFull.join("\n\n=====\n\n"));
```
Comment (revilowaldow) [inline test/test-brew.js:25]:
```suggestion
		if (!process.env.CI) fs.writeFileSync(`test/test-brew.error.log`, errorsFull.join("\n\n=====\n\n"));
```

Comment (revilowaldow) [reply test/test-brew.js]: And that's how we make a 🤏 edit from a phone.

**Comment by TheGiddyLimit** [comment]
Not a big fan of this, as it's just more copy-paste of an existing script, which the `5etools-utils` package is trying to avoid/reduce

Have updated said package with the following command, instead: `test-json-brew [file] [--dir <dir>]`

For this case, `npm i 5etools-utils@0.7.1` -> `test-json-brew --dir homebrew` while in your locally-checked-out 5etools. One could also `npm i --global` it if one was an absolute psycho, too.

Same caveats apply w.r.t. not checking the index file (I just made it ignore all `index.json`s for now, as there shouldn't be any meaningful files with that name in brew), and the logging spits out to a meh-named file in the cwd (a `.gitignore` change for that will be in next 5etools release, but feel free to add `*.log` to your local for now)

**Comment by revilowaldow** [comment]
Seems like a much better solution to me, cheers for that 👏

**Comment by revilowaldow** [comment]
image

Any guidance? :(

**Comment by TheGiddyLimit** [comment]
as resolved in the Discord thread, turns out it needs to be `npx test-json-brew --dir homebrew`

**Pull request closed without merging**